extensions/ManiphestExcelDefaultIncludeCustomFieldsFormat.php

```diff
@@ -25,35 +24,35 @@
  *   does seem to have some methods for setting/requiring a "viewer".
  *   This was not needed in the moment so it's ignored for the time being.
  */
-
-final class ManiphestExcelDefaultIncludeCustomFieldsFormat extends ManiphestExcelFormat {
+final class ManiphestExcelDefaultIncludeCustomFieldsFormatExtended extends ManiphestExcelFormat {
   public function getName() {
-    return pht('Default with Custom Fields');
+    return pht('Default with Custom Fields - extended');
```

Comment:
Sure! In reality in my installation I kept the original and added this as new file (and for this reason I changed name)